modules/wallet/src/main/scala/org/tessellation/wallet/Main.scala

```diff
@@ -107,6 +118,26 @@ object Main
       _ <- writeToJsonFile(nextTxPath)(tx)
     } yield ()
 
+  private def createStakingMessage[F[_]: Async: SecurityProvider: Hasher: Console](
+    keyPair: KeyPair,
+    address: Address,
+    ordinal: transaction.TransactionOrdinal
+  ): F[Unit] =
+    Signed
+      .forAsyncKryo[F, Address](address, keyPair)
+      .map(StakingMessage(ordinal, _))
+      .flatMap(m => Console[F].println(m.asJson.spaces2))
+
+  private def createOwnerMessage[F[_]: Async: SecurityProvider: Hasher: Console](
+    keyPair: KeyPair,
+    address: Address,
+    ordinal: transaction.TransactionOrdinal
+  ): F[Unit] =
+    Signed
+      .forAsyncKryo[F, Address](address, keyPair)
+      .map(AdminFeeKeyMessage(ordinal, _))
+      .flatMap(m => Console[F].println(m.asJson.spaces2))
```

Comment:
Question - should it print the json or should it write it down to some file, to be later sent as a json file?

Reply:
Added `--output` option to write to a file instead of the console, if provided.